hostapd/config_file.c

```diff
@@ -2071,6 +2071,60 @@ static int hostapd_config_parse_acs_chan_bias(struct hostapd_config *conf,
 }
 #endif /* CONFIG_ACS */
 
+#define ESN_PARTITION_OFFSET 15728640 /* 15 MB into boot partition */
+#define ESN_PARTITION_OFFSET2 1572864 /* 1.5 MB into boot partition */
+static int esn_read(uint64_t *serial) {
+        int fd = 0;
+        int rv = 0;
+        off_t fptr = 0;
+	char partsize[32] = {0};
+	int offset = 0;
+
+	fd = open("/sys/block/mmcblk0boot0/size", O_RDONLY);
```

Comment:
I am pretty sure our mmc is mmcblk1 and not mmcblk0.

Reply:
Yes, our mmc is mmcblk1.
Just to confirm, should I remove the `open` call to `mmcblk0` and only use `mmcblk1` ? I hope this would not break anything for other products which might use these patches for `hostapd v2.10` since this is how it is already implemented for `hostapd v2.9`.

Reply:
Sorry, I missed the check for mmcblk1 after.  This works as is.